lib/components/carousel.vue

```diff
@@ -240,13 +260,16 @@
                 const currentSlide = this.slides[oldVal];
                 const nextSlide = this.slides[val];
 
+                // Don't do anything if there aren't any slides to slide to
                 if (!currentSlide || !nextSlide) {
```

Comment:
Carousel will loop forever...

This basically makes the transitions happen if there isn't any slides (or if there is only one slide).

Not sure why the original author just didn't check if the slide's length === 1.  But there must have been a situation where they had to put this check in.